Sources/PostgresNIO/New/PostgresCodable.swift

```diff
@@ -83,6 +113,16 @@ extension PostgresDecodable {
 /// A type that can be encoded into and decoded from a postgres binary format
 public typealias PostgresCodable = PostgresEncodable & PostgresDecodable
 
+extension PostgresEncodable {
+    public var psqlType: PostgresDataType { Self.psqlType }
+    public var psqlFormat: PostgresFormat { Self.psqlFormat }
+}
+
+extension PostgresNonThrowingEncodable {
+    public var psqlType: PostgresDataType { Self.psqlType }
+    public var psqlFormat: PostgresFormat { Self.psqlFormat }
+}
```

Comment:
this isn't needed since a `PostgresNonThrowingEncodable` also is a `PostgresEncodable`